### DISCUSSION TITLE
How to explicit name the KadaluStorage in a PVC?

### DISCUSSION THREAD

**Discussion by hans-fischer** [kadalu/kadalu#461] [Q&A]
Example:
I have two storages. One is HDD other is SSD and both are of type Replica2. 

Question:
How can I create a PVC that askes for SSD storage?

**Comment by aravindavk** [comment]
Create two Storage classes by specifying the storage_name.

Refer this doc for creating custom Storage classes.

https://kadalu.io/docs/k8s-storage/latest/storage-classes

**Reply by amarts** [reply]
I tried to see if we have any mechanism to pass a key=value pair to PVC yaml. Currently couldn't find any such things, hence the above method is the only solution i can think of.

**Reply by amarts** [reply]
Should we have 'storage.replicaX' storage config at all? Looks like we can manage without any such things, and handle it similar to external config. The reason we started with Replica1 as a standard one initially was because we didn't support distribute. Now that we have that support, one volume can have one storage-config.... That way, people will be very clear what they need to refer to during PVC. @aravindavk what do you think?

**Reply by aravindavk** [reply]
Default storage classes created for each Volume types(Subvolume types). But users are not limited with these storage classes.

**Reply by aravindavk** [reply]
We can remove the default Storage classes if it is confusing.
